gnome-extra/authenticator/authenticator-9999.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+inherit gnome2-utils meson python-r1 virtualx xdg-utils
+
+DESCRIPTION="Two-factor authentication code generator for GNOME"
+HOMEPAGE="https://github.com/bilelmoussaoui/Authenticator"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	SRC_URI=""
+	EGIT_REPO_URI="${HOMEPAGE}"
+else
+	SRC_URI="https://github.com/bilelmoussaoui/Authenticator/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/Authenticator-${PV}"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-2"
```

Comment:
https://github.com/bilelmoussaoui/Authenticator/blob/master/LICENSE looks like it's `GPL-2+` not just `GPL-2`